README.md

```diff
@@ -1,2 +1,83 @@
 # test-automation
-Tools for making our tests easier to run
+
+Tools for making our tests easier to run. Automates setting up a cluster with
+CloudFormation and installs a script which automates setting up citus and everything
+required for testing citus.
+
+# Starting a cluster
+
+You'll need to have installed the (https://aws.amazon.com/cli/)[AWS CLI]. Once that's
```

Comment:
I think everyone has `ssh-agent` installed. And if we get Windows developers in the future `ssh-agent` will be the least of their problems :)

I'll look at the broken link and potentially inlining some instructions though.